src/test/ui/coherence/coherence-impls-copy.old.stderr

```diff
@@ -60,7 +60,7 @@ error[E0117]: only traits defined in the current crate can be implemented for ar
 LL | impl Copy for (MyType, MyType) {}
    | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ impl doesn't use types inside crate
    |
-   = note: the impl does not reference only types defined in this crate
+   = note: `(MyType, MyType)` is not defined in the current create
```

Comment:
This should just be `MyType`.

Reply:
I'm not sure why this error is emitted at all, I think it might be due to the tuple? I don't know how we'd change the wording to be more accurate.

Reply:
If it's possible to impl for `MyType`, it should also be possible to impl for `(MyType, MyType)`, so I think we should emit the types that are causing the root problem.

Reply:
Implementing a [trait external to the current crate on a tuple, array or slice of a type from the current crate](https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=11286201c62ece9740021b897be39510) is not ever allowed. I can extend the output to make that clearer though.